### ISSUE TITLE
Error when scrolling: Cannot read property 'offsetTop' of null

### ISSUE THREAD

**Issue by JohnAlbin** [makotot/scrollspy#8]
First, great utility! Works great.

But it is filling my browser console with the same repeated error each time I scroll.

Here's what Chrome browser (v54) is reporting:
```
Uncaught TypeError: Cannot read property 'offsetTop' of null
    at ScrollSpy.isInView (http://kss-node.local/kss-assets/scrollspy.js:111:15)
    at ScrollSpy.getElemsViewState (http://kss-node.local/kss-assets/scrollspy.js:90:23)
    at ScrollSpy.spy (http://kss-node.local/kss-assets/scrollspy.js:74:20)
    at http://kss-node.local/kss-assets/scrollspy.js:68:13
```

**Comment by makotot** [comment]
@JohnAlbin Thanks for reporting the issue :)
Is this issue could reproduce on kss-node's repo ?
I guess this can be fixed by using `getBoundingClientRect` instead of `offsetTop`, but cannot reproduce same problem on scrollspy's [demo](https://github.com/makotot/scrollspy/blob/master/index.html).

**Comment by JohnAlbin** [comment]
> but cannot reproduce same problem on scrollspy's demo.

Confirmed.

Guess I'll have to fix it myself then. :) I'll report back when I've fixed it. Thanks for the tip.

**Comment by makotot** [comment]
> I'll report back when I've fixed it.

👍 
Anyway, I have just released v0.1.3 using `getBoudingClientRect` instead of `offsetTop` .

**Comment by JohnAlbin** [comment]
Thanks, @makotot! I found the problem; bug on my end. :-p

I've upgraded to 0.1.3 and it works great! Thanks again.
